### ISSUE TITLE
Error on build

### ISSUE THREAD

**Issue by obihill** [drdk/dr-font-support#1]
Node version 0.12.7, NPM version 2.13.1 on Mac OS X Yosemite (10.10.2).

```
TTFPATCH - node.js porting v1.0 - (C) by Wolfram Esser [wok@derwok.de] 2001-02-23


- Opened: test-webfont.ttf
- Fileformat version: hex: '00010000'
- Number of Infotables: 10
- Found 'OS/2' table
- Tableoffset: hex:'00000128'
- Curret fsType: hex:'222f'
       1: reserved - not to be used, must be zero!
       2: embedding restricted (not allowed, at all!)
       4: embedding for preview & printing allowed
       8: embedding for editing allowed
- Wanted fsType: hex:'0000'
       0: embedding for permanent installation allowed
- TTF filesize: '1372' bytes
- OK: read: '1372' bytes from file

- OK: written: '1372' bytes to file

/Users/obihill/Downloads/dr-font-support-master/node_modules/dr-webfont-inliner/index.js:67
            content = "data:" + mimes[result.type] + ";base64," + result.data.toString(
                                                                             ^
TypeError: Cannot read property 'toString' of undefined
    at /Users/obihill/Downloads/dr-font-support-master/node_modules/dr-webfont-inliner/index.js:67:70
    at /Users/obihill/Downloads/dr-font-support-master/node_modules/async/lib/async.js:589:17
    at done (/Users/obihill/Downloads/dr-font-support-master/node_modules/async/lib/async.js:132:19)
    at /Users/obihill/Downloads/dr-font-support-master/node_modules/async/lib/async.js:32:16
    at /Users/obihill/Downloads/dr-font-support-master/node_modules/async/lib/async.js:586:21
    at /Users/obihill/Downloads/dr-font-support-master/node_modules/dr-webfont-inliner/index.js:46:6
    at fs.js:334:14
    at FSReqWrap.oncomplete (fs.js:95:15)
```

**Comment by phloe** [comment]
Yeah - building manually is borked as I couldn't find tooling to automate the generation of the WOFF2 file (the missing file is causing the error). I should've checked in the file I had when I published to npm... duh.

If all you need is the actual working (built) font testing script just use the one from npm: `npm install dr-font-support` :)

**Comment by phloe** [comment]
Should be fixed in v0.3.6...
